wger/utils/language.py

```diff
@@ -99,7 +100,7 @@ def load_ingredient_languages(request):
     """
 
     language = load_language()
-    languages = load_item_languages(LanguageConfig.SHOW_ITEM_INGREDIENTS)
+    languages = load_item_languages('2')
```

Comment:
Mhh, having magic numbers (or strings) is really not a good idea. We could move SHOW_ITEM_INGREDIENTS to utils/constants.py, that would probably break the circular import. Having the import inside the function is OK if it's necessary, but better write a short comment why.

The whole LanguageConfig for ingredients will ideally go away when the Open Food Facts issue is done and the DB has more products (the config is used so that if e.g. a language has only a handful of exercises, you can configure the application to also show those users exercises in english and the application is more useful)